pypit/arqa.py

```diff
@@ -739,7 +736,10 @@ def slit_profile(slf, mstrace, model, lordloc, rordloc, msordloc, textplt="Slit"
                 pgtxt = ", page {0:d}/{1:d}".format(i+1, len(pages))
             f.suptitle(desc + pgtxt, y=1.02, size=16)
         f.tight_layout()
-        slf._qa.savefig(dpi=200, orientation='landscape', bbox_inches='tight')
+        outfile = outroot+'{:03d}.png'.format(i)
+        plt.savefig(outfile, dpi=200)
+        plt.close()
+        #slf._qa.savefig(dpi=200, orientation='landscape', bbox_inches='tight')
```

Comment:
like the previous comment, I think we can search this file and remove all instances of `slf._qa.blah`